Gesture_control.py

```diff
@@ -108,43 +130,71 @@ def performAction(hand_motion):
         pyautogui.press('down')
 
 
+def playVideo():
+    global IS_VIDEO_PLAYING
+    IS_VIDEO_PLAYING = True
+    pyautogui.press('space')
```

Comment:
Define constants for these actions like key_forward, key_pause, key_reverse, key_volume_up, etc., because that may differ for different media players. Define these constants on the top and use them in press functions.